app/main.py

```diff
@@ -2,5 +2,17 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cached_info = {}
+
+    def inner(*args, **kwargs) -> int:
+        # Щоб унеможливити зміну ключа кешу, довелося перетворити
+        # ключові аргументи у tuple
```

Comment:
Could try this one! But you should remove your comments and just create additional variable for sorted kwargs tuple to make it more readable

Reply:
Done!